styles/camunda/docsInstance/hrefDocsToDocs.yml

```diff
@@ -0,0 +1,7 @@
+extends: existence
+message: "Improper link format: '%s'. Please specify the file extension."
+level: warning
```

Comment:
I'm setting these as warnings, not errors, because I am again not 100% confident that they will _always_ be violations.